### PR TITLE
Bring Travis and AppVeyor configurations in sync with the GLib one

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
@GuillaumeGomez all green

File: .travis.yml [hunk 1]
```diff
@@ -53,11 +54,12 @@ script:
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
     make regen_check;
     fi
-  - cargo test --features embed-lgpl-docs
+  - cargo doc --features "dox,embed-lgpl-docs"
+  - cargo test --features "$FEATURES,embed-lgpl-docs"
   # catch any sneaked in lgpl docs
-  - cargo build --features purge-lgpl-docs
-  - cargo doc --features "dox"
```
Comment (EPashkin) [inline .travis.yml]: This also still needed

Comment (EPashkin) [reply .travis.yml]: Oh, If you checked that `cargo doc --features "dox,embed-lgpl-docs"` adds docs,
then `embed-lgpl-docs` maybe don't needed for test

**Comment by sdroege** [comment]
All green here now @GuillaumeGomez

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**